service/waterfall.go

```diff
@@ -531,6 +590,47 @@ func countOnPreviousPage(skip int, numVersionElements int,
 	}
 }
 
+func waterfallDataAdaptor(
+	vvData versionVariantData,
+	project *model.Project,
+	skip int,
+	variantQuery string,
+) (waterfallData, error) {
+	var err error
+	finalData := waterfallData{}
+	var wfv waterfallVersions = vvData.Versions
```

Comment:
This is kind of strange, because it doesn't work. It says `[]watervallVersion` doesn't implement `Sort` interface, however `waterfallVersionS` type is defined as `[]waterfallVersion`.